File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>
#include <numeric>
#include <math.h>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    return accumulate(num_list.begin(), num_list.end(), 1, multiplies<int>()) < pow(accumulate(num_list.begin(), num_list.end(),0), 2) ? 1 : 0;
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int sum = 0;
        int mult = 1;
        
        for (int num : num_list) {
            sum += num;
            mult *= num;
        }
        
        return mult < sum * sum ? 1 : 0;
    }
}